src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -109,7 +119,7 @@ public void updateUser(User user) {
     persistentStorageAgent.writeThrough(user);
   }
 
-  /** Return true if the given username is known to the application. */
+  /** Return true if the given username is known to the application (both user list and admins list). */
```

Comment:
I think the added part of the comment here is not longer relevant.

Reply:
Thank you for pointing that out! Fixed.